app/assets/javascripts/discourse/app/lib/avatar-flair.js

```diff
@@ -52,7 +52,7 @@ function initializeAutoGroupFlair(site) {
     "trust_level_3",
     "trust_level_4",
   ].forEach((groupName) => {
-    const group = site.groups.findBy("name", groupName);
+    const group = site.groups?.findBy("name", groupName);
```

Comment:
It's a tiny "optimization", but if there are no groups available, then we could skip the whole loop, right?